core-services/prow/02_config/_config.yaml

```diff
@@ -1785,6 +1784,7 @@ tide:
     - openshift/community
     - openshift/config
     - openshift/configure-alertmanager-operator
+    - openshift/console
```

Comment:
This change discards pretty much all OCP process-related (includinng the ones not related to BZs, like cherry-pick-approved ones) label requirements on *all* branches (including past release ones), which goes way beyong the intention stated in the PR description.

Reply:
I thnk you need to do the following:

1. Remove openshift/console from the query that only covers `master` branch (you did this correclty in this PR)
2. Add an entirely new query that only covers `master` branch of `openshift/console` repo. In this PR you added the repo to an existing, relaxed-criteria query which is incorrect, I think you will need to add an entirely new query (basically make a copy of the one from where you are removing it, and 1) remove all other repos from it, 2) remove the valid-bz requirement from it).

Also, what are the expectattions when we do the cut? What criteria will hold for the `release-4.7` branch?

Reply:
We should retain all other process related controls for the branch and only remove the requirement for `valid-bug` labels post feature freeze on the master branch. Z-stream branches would still require bugs and the `valid-bug` label and we should follow the bug titling format for master even when not enforcing the label :+1: